CMakeLists.txt

```
@@ -1625,6 +1625,8 @@ hpx_check_for_mm_prefetch(DEFINITIONS HPX_HAVE_MM_PREFETCH)
 
 hpx_check_for_stable_inplace_merge(DEFINITIONS HPX_HAVE_STABLE_INPLACE_MERGE)
 
+hpx_check_for_nameable_threads(DEFINITIONS HPX_WITH_NAMEABLE_THREADS)
```

We have the convention that CMake variables are named `HPX_WITH_...` while the corresponding preprocessor constant should be named `HPX_HAVE_...` Thus this should be:
```suggestion
hpx_check_for_nameable_threads(DEFINITIONS HPX_HAVE_NAMEABLE_THREADS)
```